images/linux/scripts/installers/python.sh

```diff
@@ -11,13 +11,10 @@ source $HELPER_SCRIPTS/os.sh
 
 # Install Python, Python 3, pip, pip3
 if isUbuntu16 || isUbuntu18; then
-    apt-get install -y --no-install-recommends python python-dev python-pip python3 python3-dev python3-pip python3-venv
```

Comment:
Why do we remove Python 3 packages?

Reply:
@maxim-lobanov we don't remove them. I moved the installation to the line 17 since we don't need any condition for python3